csvw/csvw_test.go

```diff
@@ -31,12 +31,49 @@ func TestNew(t *testing.T) {
 		}
 
 		Convey("When the New csvw function is called", func() {
-			csvw := New(m, fileURL, externalPrefixURL, filterOutputID, downloadServiceURL, isCustom)
+			csvw := New(m, fileURL, externalPrefixURL, filterOutputID, downloadServiceURL)
 
 			Convey("Then the values should be set to the expected fields", func() {
 				So(csvw.Context, ShouldEqual, "http://www.w3.org/ns/csvw")
 				So(csvw.Title, ShouldEqual, m.Title)
 				So(csvw.Description, ShouldEqual, m.Description)
+
+			})
+		})
+	})
+}
+
+func TestNewCustom(t *testing.T) {
+	var customTitle = "Label 1 and Label 2"
+	var customURL = "download-service-url/downloads/filter-outputs/filter-output-id.csvw"
```

Comment:
Maybe move this to the inner convey where it is used? As it is specific to that one I think it makes sense having its scope reduced. We could also use `:=` syntax and potentially rename them to "expectedTitle" and "expctedURL"